usr/libexec/playtron/hardware-test-tool

```diff
@@ -42,6 +45,23 @@ class PlaceholderTest:
             self.done = True
 
 
+class ExternalDisplayTest(PlaceholderTest):
+    def __init__(self):
+        self.name = 'External Display|外接显示器'
+        self.reset()
+
+    def start(self, screen, font):
+        self.reset()
+
+        # `-left 1921` places the video off the edge of the main screen
+        # so the video becomes fullscreen on the secondary display
```

Comment:
It might be nice to include a comment with the link to where we originally found the video in case (1) we need a different video or (2) historical purposes if nothing else.

Reply:
Addressed.